service/scaladsl/server/src/main/resources/reference.conf

```diff
@@ -0,0 +1,10 @@
+lagom.logging {
```

Comment:
I'm a little surprised not to see some notion of "service router" or "http" in this API. It sort of looks like this sets the general lagom logging level to WARN, and the levels for certain exceptions wherever they are logged.

Reply:
I made this as a proof of concept. I am not sure on how to call these settings yet. But I will add it to the todo list